pkg/graph/schema.graphql

```diff
@@ -1289,7 +1289,10 @@ type OperationalSolution {
     pocEmail: String
     mustStartDts: Time
     mustFinishDts: Time
+    isOther: Boolean

```

Comment:
Should this be `Boolean!`? Looks like both Operational Solutions and Possible Operational Solutions have a `isOther`

Reply:
Good call! Thanks Clay! I updated this.